docs/ios/6x/automatic-instrumentation/hang-detection.md

```diff
@@ -119,213 +93,85 @@ do {
 
-Hang detection integrates seamlessly with other Embrace features:
+Hangs are automatically correlated with:
 
-- **Session Association**: Hangs are linked to the session in which they occurred
-- **User Identification**: Hangs are associated with the affected user
-- **View Tracking**: Correlate hangs with the active view controller or SwiftUI view
-- **Network Monitoring**: Identify if network requests coincide with hang events
-- **Crash Reporting**: Hangs that occur immediately before crashes provide critical context
+- Sessions and user identification
+- Active views
+- Network requests
+- Crashes
 
 ## Best Practices
 
-### Production Settings
-
-For production apps, the default remote configuration provides conservative settings optimized for minimal overhead:
-
-- `hangPerSession: 200` - Reasonable limit for most apps
-- `samplesPerHang: 0` - Captures initial stack trace only, minimizing overhead
+### Default Settings
 
-This configuration:
+The default configuration (`hangPerSession: 200`, `samplesPerHang: 0`) is optimized for production with minimal overhead.
 
-- Captures hang occurrence, duration, and the initial blocking stack trace
-- Minimizes performance impact
-- Provides sufficient data for identifying problematic areas
+### Debugging
 
-:::info
-These settings are managed automatically via Embrace's remote configuration. You typically don't need to customize them in your code.
-:::
-
-### Debugging Settings
-
-When actively debugging hang issues, you can increase sampling via remote configuration or by implementing a custom configuration for development builds:
-
-```swift
-class DebugHangConfig: EmbraceConfigurable {
-    var hangLimits = HangLimits(
-        hangPerSession: 200,
-        samplesPerHang: 10      // Detailed stack trace sampling
-    )
-
-    // ... other required EmbraceConfigurable properties with defaults
-}
-```
-
-This provides:
-
-- Multiple stack snapshots throughout each hang (up to 10 samples)
-- Better visibility into what code is blocking the main thread and how it evolves
-- More context for reproducing and fixing issues
-
-:::caution Performance Impact
-Stack trace sampling requires briefly suspending the main thread, which adds minimal overhead but should be used judiciously in production. Consider enabling detailed sampling for a small percentage of users or in pre-production builds.
-:::
-
-### Interpreting Hang Data
-
-When analyzing hang data in the Embrace dashboard:
-
-1. **Look for Patterns**: Are hangs concentrated in specific views or features?
-2. **Check Duration**: Short hangs (`<500ms`) vs. long hangs (`>1s`) may have different causes
-3. **Review Stack Traces**: Identify the common blocking operations
-4. **Correlate with Sessions**: Do hangs occur more frequently in certain user segments?
-5. **Monitor Frequency**: Track whether fixes reduce hang occurrence
+When debugging hang issues, increase `samplesPerHang` to 5-10 for more detailed stack traces.
 
 ### Common Hang Sources
```

Comment:
Interesting to have, but I'm not sure the example can relate too much to customers. Maybe worth revisiting later and removing it for now.

Reply:
Let's give users a link to Apple docs on Hangs instead and keep our docs to what Embrace does for hangs.

https://developer.apple.com/documentation/xcode/understanding-hangs-in-your-app

https://developer.apple.com/documentation/xcode/improving-app-responsiveness